locale/ru/LC_MESSAGES/reference/reference_lua/net_box.po

```diff
@@ -54,7 +54,7 @@ msgid ""
 "network socket, but each fiber gets back a correct response. Reducing the "
 "number of active sockets lowers the overhead of system calls and increases "
 "the overall server performance. However for some cases a single connection "
-"is not enough---for example, when it is necessary to prioritize requests or "
+"is not enough -- for example, when it is necessary to prioritize requests or "
```

Comment:
Are we start to add spaces around a dash in this case? Should we give certain rule in the style guide?

Reply:
The style guide provides for it. See line 200 in `style.rst`: "Add spaces on both sides of the dash."